Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Functions.cs

```diff
@@ -20,19 +22,24 @@ public Functions()
     /// <summary>
     /// A Lambda function to respond to HTTP Get methods from API Gateway
     /// </summary>
-    /// <param name="request"></param>
-    /// <returns>The API Gateway response.</returns>
-    public APIGatewayProxyResponse Get(APIGatewayProxyRequest request, ILambdaContext context)
+    /// <remarks>
+    /// This uses the <see href="https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md">Lambda Annotations</see> 
+    /// programming model to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+    /// 
+    /// This automatically handles reading parameters from an APIGatewayProxyRequest
```

Comment:
nit: could you add a doc reference to APIGatewayProxyRequest?

Reply:
Done